package Prueba;

public class NullPointerException {
    public static void main(String[] args) {



      try {
           

        } catch (Exception e) {
            //flujo aterno
  
            // TODO: handle exception
            System.out.println( "division entre cero");
        } finally { 
           System.out.println("Gracias por utlizar el programa");
        
        }
    }
}    